dll/win32/shlwapi/CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
 
 project(SHELL)
 
-remove_definitions(-D_WIN32_WINNT=0x502)
-add_definitions(-D_WIN32_WINNT=0x600)
-
```

Comment:
I do like your current approach of encapsulating that existing hack to an even more isolated/distinct part of shlwapi (assoc.c).
Can we split off that part from your PR5511 "SHCreatePropertyBagOnMemory" and commit that on its own?
I think it is a decent improvement in itself already.

Reply:
https://github.com/reactos/reactos/pull/5514